ChiModules/LinearBoltzmannSolver/lbs_structs.h

```diff
@@ -28,7 +28,7 @@ struct Options
   GeometryType geometry_type = GeometryType::NO_GEOMETRY_SET;
   SDMType sd_type = SDMType::UNDEFINED;
   unsigned int scattering_order=1;
-  int  sweep_eager_limit= 32000;;
+  int  sweep_eager_limit= 32000;
```

Comment:
I don't think this bit of code was introduced in this PR, but we should document why we chose this value.  In particular, it should probably be a power of two... especially if it is supposed to correspond to the MPI eager limit or indicate a buffer size.

Reply:
@zachhardy Zach can you add just abbreviated comment here? The input documentation for `chiLBSSetProperty` has this documented well but we probably need to add an abbreviated version here and reference the documentation.

Reply:
@Naktakala Added a blurb that references the documentation in chiLBSSetProperty.